### PR TITLE
get GMM weight

### PR DESCRIPTION
**Pull request by junpenglao**
save GMM model mixture weight and read out from checkpoint file

### CODE REVIEW

**Comment by junpenglao** [comment]
I signed it!

File: tensorflow/contrib/factorization/python/ops/gmm.py [hunk 1]
```diff
@@ -102,7 +102,12 @@ def score(self, input_fn=None, batch_size=None, steps=None):
     results = self.evaluate(input_fn=input_fn, batch_size=batch_size,
                             steps=steps)
     return np.sum(results[GMM.SCORES])
-
+  
+  def weights(self):
+    """Returns the cluster weights."""
+    return checkpoint_utils.load_variable(
+        self.model_dir, gmm_ops.GmmAlgorithm.CLUSTERS_WEIGHT)
```
Comment (rmlarsen) [inline tensorflow/contrib/factorization/python/ops/gmm.py]: Is there a unit test for this?

**Comment by rmlarsen** [comment]
@tensorflow-jenkins test this please

File: tensorflow/contrib/factorization/python/ops/gmm.py (hunk 1 above)
Comment (junpenglao) [inline tensorflow/contrib/factorization/python/ops/gmm.py]: not yet but i can put one in [gmm_test.py](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/contrib/factorization/python/ops/gmm_test.py)

Comment (rmlarsen) [inline tensorflow/contrib/factorization/python/ops/gmm.py]: @junpenglao thanks, please to.

**Comment by rmlarsen** [comment]
@tensorflow-jenkins test this please

**Review by rmlarsen — APPROVED** [review]

**Comment by rmlarsen** [comment]
The filing saver_test is unrelated. Merging.

**Pull request merged**